spec/moneta/adapters/client/standard_client_tcp_spec.rb

```diff
@@ -4,14 +4,14 @@
   include_context :start_server, port: 9003, backend: ->{ Moneta::Adapters::Memory.new }
 
   moneta_store :Client, port: 9003
-  moneta_specs STANDARD_SPECS
+  moneta_specs STANDARD_SPECS.with_each_key
 
   it 'supports multiple clients' do
     store['shared_key'] = 'shared_val'
-    threads = (1..100).map do |i|
+    threads = (1..32).map do |i|
       Thread.new do
         client = new_store
-        (1..100).each do |j|
+        (1..31).each do |j|
```

Comment:
I'm not clear on why this change was needed.  Could you give some context?

Reply:
Not quite sure. I think that in my local (MacOS) there is some kind of limitation with the amount of open ports. 
I figure that if it works with almost 32 it should work with 100...